tinyformat.h

```diff
@@ -847,7 +851,15 @@ class FormatListN : public FormatList
         template<typename... Args>
         FormatListN(const Args&... args)
             : FormatList(&m_formatterStore[0], N),
-            m_formatterStore{FormatArg(args)...}
+            m_formatterStore
+// VS2013 needs a pair of extra braces to compile
+#ifdef _MSC_VER
```

Comment:
Perhaps it would be cleaner to define a workaround macro up near the top:

```C++
#ifdef _MSC_VER && _MSC_VER <= some_version_i_dunno
#   define TINYFORMAT_BRACED_INIT_WORKAROUND(x) (x)
#else
#   define TINYFORMAT_BRACED_INIT_WORKAROUND(x) x
#endif
```

Then this code can just use `m_formatterStore TINYFORMAT_BRACED_INIT_WORKAROUND({FormatArg(args)...})` here without the conditionals.